### PR TITLE
Custom 400,403,404&500

### PR DESCRIPTION
**Pull request by drowsy810301**
Custom 400,403,404&500 page,you can raise the exception through following code

```
from django.http import Http404 #404 page
from django.core.exceptions import PermissionDenied #403 page
from django.core.exceptions import SuspiciousOperation #400 page

raise SuspiciousOperation('your message') #400 page
raise PermissionDenied('your message') #403 page
raise Http404 #404 page
raise Exception('your message') #500 page
```

### CODE REVIEW

File: index/templates/index/400.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+SOFTWARE.
+-->
+<!DOCTYPE html>
+{% load static %}
+
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <!-- necessarily jQeuery -->
+  <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.1/jquery.min.js">
```
Comment (henryyang42) [inline index/templates/index/400.html]: Maybe these cdn link is no longer needed.

**Comment by drowsy810301** [comment]
fixed

**Comment by nickelcheng** [comment]
Why do I get a django 404 page instead of a customize 404 page?

**Comment by henryyang42** [comment]
Set ``DEBUG=False`` as the page bottom says.

**Comment by nickelcheng** [comment]
Thanks.
+1, @bruce3557 please check.

**Comment by drowsy810301** [comment]
conflict fixed

**Comment by drowsy810301** [comment]
please review again,thanks

File: index/templates/index/403.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+SOFTWARE.
+-->
+<!DOCTYPE html>
+{% load static %}
+
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <!-- bootstrap, bootwatch JS & CSS -->
+  <link rel="stylesheet" href={% static "bootswatch/paper/bootstrap.css" %}>
```
Comment (bruce3557) [inline index/templates/index/403.html]: Is the theme hardcoded?
I see there is a setting in profile to choose a theme by preference.

**Comment by bruce3557** [comment]
+1 besides the comment

**Comment by drowsy810301** [comment]
comment fixed

**Comment by nickelcheng** [comment]
going to merge.

**Pull request merged**